lib/index.js

```diff
@@ -92,6 +93,63 @@ var specialElHandlers = {
 function noop() {}
 
 /**
+ * Create an element, optionally with a known namespace URI.
+ *
+ * @param {String} name the element name, e.g. 'div' or 'svg'
+ * @param {String?} namespaceURI the element's namespace URI, i.e. the value of
+ * its `xmlns` attribute or its inferred namespace.
+ *
+ * @return {Element}
+ */
+function createElement(name, namespaceURI) {
+    return namespaceURI ?
+        document.createElementNS(namespaceURI, name) :
+        document.createElement(name);
+}
+
+/**
+ * Get an element's namespace-aware attribute value.
+ *
+ * @param {Element} el
+ * @param {Attr} attr   an Attr reference, from which we can determine the
+ * namespace URI.
+ *
+ * @return {String}
+ */
+function getAttribute(el, attr) {
+    return attr.namespaceURI ?
+        el.getAttributeNS(attr.namespaceURI, attr.name) :
+        el.getAttribute(attr.name);
+}
+
+/**
+ * Set an element's namespace-aware attribute value.
+ *
+ * @param {Element} el
+ * @param {Attr}    attr   an Attr reference, from which we can determine the
+ * namespace URI.
+ * @param {String}  value  the attribute value
+ */
+function setAttribute(el, attr, value) {
+    return attr.namespaceURI ?
+        el.setAttributeNS(attr.namespaceURI, attr.name, value) :
+        el.setAttribute(attr.name, value);
+}
+
+/**
+ * Set an element's namespace-aware attribute.
+ *
+ * @param {Element} el
+ * @param {Attr}    attr   an Attr reference, from which we can determine the
+ * namespace URI.
+ */
+function removeAttribute(el, attr) {
```

Comment:
I'm not opposed to dropping support for IE8 (we dropped support for IE8 at eBay and that is my primary concern), but we would just need to increment the major version and update the docs in case there are any others that need IE8 support (which I suspect is unlikely).